generators/server/templates/src/main/resources/mails/creationEmail.html

```diff
@@ -12,8 +12,8 @@
             Your JHipster account has been created, please click on the URL below to access it:
         </p>
         <p>
-          <a th:href="@{|${baseUrl}/#/reset/finish?key=${user.resetKey}|}"
```

Comment:
This is set up this way for a reason - the user is already activated and receives an email to set a password.